src/main/java/persistence/sql/meta/ColumnLength.java

```diff
@@ -0,0 +1,43 @@
+package persistence.sql.meta;
+
+import jakarta.persistence.Column;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.Objects;
+
+public class ColumnLength {
+    private static final int DEFAULT_LENGTH = 0;
+
+    private final List<Class<?>> lengthNeededTypes = List.of(String.class);
+    private final int length;
```

Comment:
`ColumnLength.getLength()`

과한 일반화일수 있겠지만, `Length` 두번반복되는게 눈에 보여서... 
이미 `ColumnLength`라는 이름으로 강력하게 표현해준것 같아서  `ColumnLength.value()` 라는것도 떠오르긴하네요.
반영을 위한 코멘트는 아니에요~ ㅎ

```suggestion
    private static final List<Class<?>> lengthNeededTypes = List.of(String.class);
    private final int length;

    ...
    
    public int value() {
        return length;
    }
```

Reply:
좋은 의견이신 것 같아요.
중복 표현으로 신경이 쓰였던 부분이긴 했었어요 😄